manifest/product-config.json

```diff
@@ -129,6 +129,14 @@
             "go_version": "1.11.5",
             "start_build": 270
         },
+        "manifest/2.6.xml": {
+            "release": "2.6.0",
+            "release_name": "Couchbase Sync Gateway 2.6.0",
+            "production": true,
+            "interval": 120,
+            "go_version": "1.11.5",
+            "start_build": 128
```

Comment:
Not 100% sure about this - I've assumed it to be N+1 of the current build number.